manifests/repo.pp

```diff
@@ -1,34 +0,0 @@
-# lldpd::repo
-#
-# @api private
-#
-# @summary Manages the yum or apt repositories
-#
-class lldpd::repo {
-  assert_private()
-
-  if $lldpd::manage_repo {
-    case $facts['os']['family'] {
-      'RedHat': {
-        yumrepo { 'lldpd':
-          ensure  => 'present',
-          descr   => 'lldpd toolset',
-          gpgkey  => "https://download.opensuse.org/repositories/home:/vbernat/${lldpd::repourl}/repodata/repomd.xml.key",
-          baseurl => "https://download.opensuse.org/repositories/home:/vbernat/${lldpd::repourl}/",
-        }
-      }
-      'Debian': {
-        apt::source { 'lldpd':
```

Comment:
did this get lost or did you purge it on purpose?

Reply:
As far as i cant tell lldpd is available in stock stretch and buster.  I assumed it would be in the ubuntu varients but i havn't checked

Reply:
I added those repos because the debian repos contain pretty old lldpd packages :(

Reply:
Added back and reset the defaults.